packages/ember-simple-auth/.eslintrc.js

```diff
@@ -57,6 +57,11 @@ module.exports = {
         node: true,
         mocha: true
       }
-    }
+    },
+    {
+      // test files
+      files: ['tests/**/*-test.{js,ts}'],
+      extends: ['plugin:qunit/recommended'],
+    },
```

Comment:
See above – I assume this should go into the config that previously contained the Mocha linting settings?